include/CLI/Option.hpp

```diff
@@ -418,7 +418,7 @@ class Option : public OptionBase<Option> {
     }
 
     /// Adds a Validator. Takes a const string& and returns an error message (empty if conversion/check is okay).
-    Option *check(std::function<std::string(const std::string &)> Validator,
+    Option *check(const std::function<std::string(const std::string &)> &Validator,
                   std::string Validator_description = "",
                   std::string Validator_name = "") {
         validators_.emplace_back(Validator, std::move(Validator_description), std::move(Validator_name));
```

Comment:
wonder if Validator should be moved here instead of copied, same with a few functions down.  Whatever is done it should probably be consistent on all functions

Reply:
I agree that this should be consistent. Reverted in this PR, maybe this should be done in a separate PR